src/model.coffee

```diff
@@ -51,6 +56,13 @@ class window.Brainstem.Model extends Backbone.Model
 
 
   #
+  # Init
+
+  constructor: (options = {}) ->
+    super
+    @storageManager = StorageManager.get()
```

Comment:
We can make it a dependency similar to Backbone that's a singleton so we can avoid the circular dependencies that exist currently right?